focus-android/app/src/androidTest/java/org/mozilla/focus/activity/EnhancedTrackingProtectionSettingsTest.kt

```diff
@@ -73,7 +67,6 @@ class EnhancedTrackingProtectionSettingsTest {
         }
     }
 
-    @Ignore
     @SmokeTest
```

Comment:
Can we add a summary of the workaround in this class and link to the logged original Gecko ticket we filed?

Reply:
sure. done